include/seqan3/core/type_traits/transformation_trait_or.hpp

```diff
@@ -42,7 +58,7 @@ namespace seqan3::detail
  *   seqan3::detail::transformation_trait_or_t as a shorthand for *seqan3::detail::transformation_trait_or::type*
  */
 template <typename type_t, typename default_t>
-using transformation_trait_or = std::conditional_t<meta::is_trait<type_t>::value,   // check if type_t::type exists
+using transformation_trait_or = std::conditional_t<is_trait_v<type_t>,              // check if type_t::type exists
```

Comment:
Actually, this can be
```suggestion
using transformation_trait_or = std::conditional_t<transformation_trait<type_t>,   // check if type_t::type exists
```

And then you don't need the is_trait_v